README.md: Strongly discourage using any device besides Google Pixels

chenxiaolong opened the pull request:
OnePlus seems to be getting worse and worse with some devices not being recoverable.

Issue: #290

xabolcs commented:
For the record:

I don't know about Nothing Phone (2) and (2a), but Nothing Phone (1) - `Spacewar` works nicely with `avb_custom_key`. :pray:

chenxiaolong commented:
Thanks! I've moved the information out of the README into #299 and added a link (029cb4264e0502be0795fb78862ba567a6cc11f3). This should hopefully make it easier to keep track of the list of devices that work properly.
